KubeArmor/core/kubeArmor.go

```diff
@@ -321,7 +321,8 @@ func KubeArmor() {
 	// create a daemon
 	dm := NewKubeArmorDaemon()
 
-	if cfg.GlobalCfg.KVMAgent {
+	// Enable KubeArmorHostPolicy for both VM and KVMAgent
+	if !kl.IsK8sEnv() && (cfg.GlobalCfg.KVMAgent || cfg.GlobalCfg.HostPolicy) {
```

Comment:
What happens in cases where user has `~/.kube/config` file, but still wants to use kubearmor in pure VM/systemd mode?
Just the presence of kubeconfig file will cause kubearmor to think that it is using k8s mode of operation!

Reply:
How can we know if a user wants to run KubeArmor as a systemd mode in such as case?
@seswarrajan What happens if a user uses -enableKubeArmorVm without kvm-service?

Reply:
@nam-jaehyun User can enable KubeArmor systemd mode only by -enableKubeArmorHostPolicy flag. It is not possible to run kubeArmor in VM with -enableKubeArmorVm flag as this flag tries to connect to kvm-service and if the connection fails, KubeArmor will terminate.

Reply:
@nyrahul In that case we need to ignore the IsK8sEnv() validation check in the code.

Reply:
This has to be fixed holistically ... currently the code just allows to bypass the first hurdle. If the IsK8sEnv() check is present elsewhere, then VM/systemd mode cannot be used.

Can you check if kubearmor works in systemd mode (with policy enforcement) when there is ~/.kube/config in the path?